puddle.yml

```diff
@@ -1,5 +1,5 @@
 ---
-- hosts: puddle
+- hosts: "{{ targets | default('puddle') }}"
```

Comment:
Agreed.  This is really only an issue for playbooks that teuthology might want to run, but this isn't changing any behavior for cobbler.yml or puddle.yml though so I felt it was harmless.